bin/test.c

```diff
@@ -190,52 +190,101 @@ void test_parser() {
   printf("\n");
 }
 
+static void test_serialiser_procces_msg(mqtt_serialiser_t* serialiser, mqtt_message_t* message) {
```

Comment:
Rename the function to `test_serialiser_proccess_message`.

Reply:
> Rename the function to test_serialiser_proccess_message.

@rasdie You have marked this as resolved but I don't see the change? Can you please commit it?